BE/GovernmentEntities/AsianJurisdiction/EasternAsiaGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -107,35 +100,30 @@
 	<owl:NamedIndividual rdf:about="&fibo-be-ge-easj;GovernmentOfThePeoplesRepublicOfChina">
 		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
 		<rdfs:label xml:lang="en">Government of the People&apos;s Republic of China</rdfs:label>
+		<rdfs:label xml:lang="zh">中华人民共和国</rdfs:label>
 		<skos:definition>unitary Marxist-Leninist single-party socialist republic in Eastern Asia, covering approximately 9.6 million square kilometers (3.7 million square miles), that is the world&apos;s third or fourth-largest country by area</skos:definition>
 		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-easj;JurisdictionOfChina"/>
 		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;China"/>
 	</owl:NamedIndividual>
 	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-easj;GovernmentOfTheRepublicOfChina">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of the Republic of China</rdfs:label>
+		<rdfs:label xml:lang="zh">中華民國</rdfs:label>
+		<skos:definition>unitary semi-presidential constitutional republic whose neighbors include the People&apos;s Republic of China (PRC) to the northwest, Japan to the northeast, and the Philippines to the south</skos:definition>
```

Comment:
What does "semi-presidential" even mean? We're on the verge of needing our own ontology for forms of government (not suggesting this).
I looked it up here https://en.wikipedia.org/wiki/Semi-presidential_system and discovered there are two important subtypes. Why not use those too?
My question really is where are we going with this and based on what use case(s)? And what conventions, standards or guidelines are we setting for others when they create instances of government.
In terms of use cases for FIBO, more useful would be some sort of rating of governments by level of probity/corruption such as https://www.transparency.org/en/cpi/2020/index/nzl. Or whether they provide banking transparency, extradition or cross-border taxation. Again not suggesting we do this, but please let's not add stuff because we can, but because there's a genuine financial/FIBO use case.